pkg/render/intrusion_detection.go

```diff
@@ -430,6 +437,10 @@ func (c *intrusionDetectionComponent) intrusionDetectionJobServiceAccount() *cor
 }
 
 func (c *intrusionDetectionComponent) intrusionDetectionClusterRole() *rbacv1.ClusterRole {
+	stdApiResources := []string{"podtemplates"}
+	if c.deployWebhooksController() {
+		stdApiResources = append(stdApiResources, "secrets", "configmaps")
```

Comment:
In golang, acryonyms shoudl be same-case.  Meaning this should be `stdAPIResources`.

While here though, I think this should be renamed entirely since it's not clear what `std` means in this context. Maybe this shoudl be called `coreGetResources` since these are the resources in the core API group that need get permissions?

Reply:
Good point, I have reorganized this function a bit so this variable is no longer required.